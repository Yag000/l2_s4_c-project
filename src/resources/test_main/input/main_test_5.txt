mkdir a
cd a
pwd    
mkdir b
cd b
pwd
mkdir c
cd c
pwd
cd
pwd
mkdir a/b/c/d
pwd
mkdir a/b/c/d/e
mkdir a/b/c/d/e/f
print
mv a newa
print
mv newa/b newa/newb
mv newa/newb/c newa/newb/newc
mv newa/newb/newc/d newa/newb/newc/newd    
mv newa/newb/newc/newd/e newa/newb/newc/newd/newe        
mv newa/newb/newc/newd/newe/f newa/newb/newc/newd/newe/newf
print
cp newa copya
cp newa/newb copya/copyb
rm copya/newb
cp newa/newb/newc copya/copyb/copyc
rm copya/copyb/newc
cp newa/newb/newc/newd copya/copyb/copyc/copyd
rm copya/copyb/copyc/newd
cp newa/newb/newc/newd/newe copya/copyb/copyc/copyd/copye
rm copya/copyb/copyc/copyd/newe
cp newa/newb/newc/newd/newe/newf copya/copyb/copyc/copyd/copye/copyf
rm copya/copyb/copyc/copyd/copye/newf
print
cd newa/newb/newc/newd/newe/newf
pwd
mv /copya/copyb ./copyb
rm /copya
print
mkdir /copya
cp copyb /copya/copyb
print
rm copyb
print
cd
rm newa
rm copya
print