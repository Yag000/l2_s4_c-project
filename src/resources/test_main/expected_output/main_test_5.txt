/a
/a/b
/a/b/c
/
/
/ (D)
\-- a (D)
    \-- b (D)
        \-- c (D)
            \-- d (D)
                \-- e (D)
                    \-- f (D)
/ (D)
\-- newa (D)
    \-- b (D)
        \-- c (D)
            \-- d (D)
                \-- e (D)
                    \-- f (D)
/ (D)
\-- newa (D)
    \-- newb (D)
        \-- newc (D)
            \-- newd (D)
                \-- newe (D)
                    \-- newf (D)
/ (D)
|-- newa (D)
|   \-- newb (D)
|       \-- newc (D)
|           \-- newd (D)
|               \-- newe (D)
|                   \-- newf (D)
\-- copya (D)
    \-- copyb (D)
        \-- copyc (D)
            \-- copyd (D)
                \-- copye (D)
                    \-- copyf (D)
/newa/newb/newc/newd/newe/newf
/ (D)
\-- newa (D)
    \-- newb (D)
        \-- newc (D)
            \-- newd (D)
                \-- newe (D)
                    \-- newf (D)
                        \-- copyb (D)
                            \-- copyc (D)
                                \-- copyd (D)
                                    \-- copye (D)
                                        \-- copyf (D)
/ (D)
|-- newa (D)
|   \-- newb (D)
|       \-- newc (D)
|           \-- newd (D)
|               \-- newe (D)
|                   \-- newf (D)
|                       \-- copyb (D)
|                           \-- copyc (D)
|                               \-- copyd (D)
|                                   \-- copye (D)
|                                       \-- copyf (D)
\-- copya (D)
    \-- copyb (D)
        \-- copyc (D)
            \-- copyd (D)
                \-- copye (D)
                    \-- copyf (D)
/ (D)
|-- newa (D)
|   \-- newb (D)
|       \-- newc (D)
|           \-- newd (D)
|               \-- newe (D)
|                   \-- newf (D)
\-- copya (D)
    \-- copyb (D)
        \-- copyc (D)
            \-- copyd (D)
                \-- copye (D)
                    \-- copyf (D)
/ (D)
